client/ayon_flame/plugins/publish/collect_shots.py

```diff
@@ -0,0 +1,342 @@
+import pyblish
+import re
+from pprint import pformat
+
+import ayon_flame.api as ayfapi
+from ayon_flame.otio import flame_export
+
+from ayon_core.pipeline.editorial import (
+    get_media_range_with_retimes
+)
+
+
+# constatns
+NUM_PATTERN = re.compile(r"([0-9\.]+)")
+TXT_PATTERN = re.compile(r"([a-zA-Z]+)")
+
+
+class CollectShot(pyblish.api.InstancePlugin):
+    """Collect new shots."""
+
+    order = order = pyblish.api.CollectorOrder - 0.095
```

Comment:
```suggestion
    order = pyblish.api.CollectorOrder - 0.095
```

Reply:
Is it possible to run this plugin much earlier? e.g. `pyblish.api.CollectorOrder - 0.49`? If not please describe why not. Does not happen in this PR, but let me know I'll create issue for it if we can't do it now, thanks.

Reply:
I have changed the order to match your recommendations.